### PR TITLE
Add section to docs for creating data for cubeviz

### PR DESCRIPTION
**Pull request by javerbukh**
### Description

This pull request is to add a section to readthedocs explaining to users how to create a Spectrum1d cube from scratch and then add it to cubeviz from within a jupyter notebook instance.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [ ] Is a change log needed? If yes, is it added to `CHANGES.rst`?
- [ ] Is a milestone set?
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

File: docs/cubeviz/import_data.rst [hunk 1]
```diff
@@ -43,3 +43,24 @@ method, which takes as input either the name of a local file or a
...
+
+    flux = np.arange(16).reshape((2, 2, 4)) * u.Jy
+    wcs_dict = {"CTYPE1": "RA---TAN", "CTYPE2": "DEC--TAN", "CTYPE3": "WAVE-LOG",
+                "CRVAL1": 205, "CRVAL2": 27, "CRVAL3": 4.622e-7,
+                "CDELT1": -0.0001, "CDELT2": 0.0001, "CDELT3": 8e-11,
+                "CRPIX1": 0, "CRPIX2": 0, "CRPIX3": 0}
+    w = WCS(wcs_dict)
+
+    cube = Spectrum1D(flux=flux, wcs=w)
+    cubeviz.load_data(cube) # doctest: +SKIP
```
Comment (kecnry) [inline docs/cubeviz/import_data.rst:63]: would doctest pass in this case since its not a fake filename?

**Review by kecnry — APPROVED** [review]
Example covers all the basics while still remaining easy-to-read.  Would a link to specutils docs be useful here or does it already exist somewhere close enough?

File: docs/cubeviz/import_data.rst (hunk 1 above)
Comment (pllim) [reply docs/cubeviz/import_data.rst:63]: If you want doctest, you need to prepend the command with `>>>` and continuation with `...`.

Also, I agree with @kecnry that this should be doctested if we want to take doctest seriously.

Comment (pllim) [reply docs/cubeviz/import_data.rst:63]: p.s. But you can skip the `cubeviz.app` line.

File: docs/cubeviz/import_data.rst [hunk 2]
```diff
@@ -43,3 +43,24 @@ method, which takes as input either the name of a local file or a
...
+If instead you would like to load data you have created within the notebook,
+you can do the following::
+
+    import numpy as np
+    from astropy import units as u
+    from astropy.wcs import WCS
+    from specutils import Spectrum1D
+
+    flux = np.arange(16).reshape((2, 2, 4)) * u.Jy
+    wcs_dict = {"CTYPE1": "RA---TAN", "CTYPE2": "DEC--TAN", "CTYPE3": "WAVE-LOG",
```
Comment (rosteen) [inline docs/cubeviz/import_data.rst:56]: Just a reminder, the WCS axis order is the opposite of numpy, so if you wanted your length 4 axis to be the wavelength then `CTYPE1` should be your `WAVE-LOG`.

File: docs/cubeviz/import_data.rst (hunk 1 above)
Comment (javerbukh) [reply docs/cubeviz/import_data.rst:63]: We actually had a conversation last week about removing the `>>>` from lines in the documentation for consistencies sake. Was that a mistake @pllim ?

File: docs/cubeviz/import_data.rst (hunk 2 above)
Comment (javerbukh) [reply docs/cubeviz/import_data.rst:56]: This is straight from `jdaviz/conftest.py`, does that mean it is wrong there as well?

File: docs/cubeviz/import_data.rst (hunk 1 above)
Comment (pllim) [reply docs/cubeviz/import_data.rst:63]: If you want doctest to run, you need the `>>>`.

For example: In the matplotlib Sphinx plot directive, they don't take `>>>`, so in `astropy`, we have to write the code twice, once in the plot directive to auto-generate the plot, and the other with `>>>` so it gets tested. It is a pain but currently there is no better solution.

If you do not want to doctest the code, it does not matter.

File: docs/cubeviz/import_data.rst (hunk 2 above)
Comment (javerbukh) [reply docs/cubeviz/import_data.rst:56]: Got it.

**Comment by pllim** [comment]
@rosteen , you happy with the axis order?

**Review by rosteen — APPROVED** [review]
Yes, last time I looked the tests were still running, was just waiting for that.

**Pull request merged**